config/custom-environment-variables.json

```diff
@@ -0,0 +1,19 @@
+{
+  "session": {
+    "useRedis": {
+      "__name": "USE_REDIS",
+      "__format": "json"
+    },
+    "useLogger": "USE_SESSION_LOGGER",
+    "redis": {
```

Comment:
we do need this :) 
Please, inject these values in the session in the app.ts file like in this example: https://github.com/hmcts/ia-aip-frontend/blob/master/app/session.ts#L18